src/common/constants.ts

```diff
@@ -1,5 +1,10 @@
 import { BigNumber } from '0x.js';
 
+export enum MARKETPLACES {
+    ERC20 = 'ERC20',
+    ERC721 = 'ERC721',
+}
+
```

Comment:
Req: I think this enum should not be here, Can we move it to the `util / types` ?

Reply:
Done